eazieda/eazieda.py

```diff
@@ -119,16 +127,20 @@ def missing_imputed(data, impute=False, method_num="mean", method_non_num="most_
     impute : bool, default = False
         Whether to impute the missing values in place.
     method_num : str, default = "mean"
-        The method used for imputing numerical missing values. This is only applicable if impute=True.
+        The method used for imputing numerical missing values. 
```

Comment:
Let's get rid of . at the end of docsrtings

Reply:
I think this is useful information - let's keep in for this week?